tests/testthat/test-plot_sankey.R

```diff
@@ -0,0 +1,12 @@
+test_that("returns an object of the expected class", {
+  p <- plot_sankey(toy_data)
+  expect_s3_class(p, "sankeyNetwork")
+  expect_s3_class(p, "htmlwidget")
+})
```

Comment:
You may test expected properties of the resulting plot like this:

``` r
devtools::load_all()
#> ℹ Loading tiltPlot

p <- plot_sankey(toy_data)
p$x$nodes$name
#> [1] "tilt_sec" "A"        "B"        "low"      "high"
p$x$nodes$group
#> [1] "other" "other" "other" "low"   "high"
```

<sup>Created on 2023-06-14 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>